synapse/api/constants.py

```diff
@@ -34,7 +34,7 @@ class Membership:
 
     INVITE = "invite"
     JOIN = "join"
-    KNOCK = "knock"
+    KNOCK = "xyz.amorgan.knock"
```

Comment:
I assume this is fine? Nothing else will be using `KNOCK` in the meantime, and if something *does* end up needing it then a separate constant can be created for the unstable feature.

Reply:
I think so? It might make more sense to call this `UNSTABLE_KNOCK` and have a separate `KNOCK`? Either way seems OK to me.